Project-Bootcamp/querys/query.js

```diff
@@ -0,0 +1,66 @@
+var $ = require ('jquery');
+var _ = require ('lodash');
+
+var Apikey = 'qrmacznjya3vdnynqdr4trf3bd9t3fau';
```

Comment:
No need to define this here, you can set ``this.apiKey`` directly on line 9 (should be line 8, LINE BREAKS! :P). Also, it's not a class or component, so proper grammar should be *apiKey* (or at least *API_KEY* if we're following a coding styling standard that defines constants like that).